capstone-app/src/app/modules/history/history.html

```diff
@@ -0,0 +1 @@
+
```

Comment:
This file appears to only be a newline. Was this going to be used for something? If not, then delete it.